tests/suites/test_suite_mpi.function

```diff
@@ -765,16 +765,20 @@ exit:
 
 /* BEGIN_CASE */
 void mpi_core_cond_assign( data_t * input_X,
-                           data_t * input_Y )
+                           data_t * input_Y,
+                           int input_len,
+                           int is_fail )
```

Comment:
Blocker: I don't understand the point of `input_len` and `copy_limbs`. Why does it matter that what's being assigned is only a prefix of the allocated buffer? And what is `is_fail`, given that the tested function has no failure mode?

This needs at least some documentation.

Goes for swap as well.

Reply:
The `input_len` is needed to test that the assign function is always copying the given length even it is smaller than the length of the given MPIs. As I think about it again the `is_fail` is not needed.
I will add a comment to explain it.